### ISSUE TITLE
Required fields should be indicated as such in markup

### ISSUE THREAD

**Issue by aarongustafson** [18F/web-design-standards#750]
I’m happy to run a pull request on this, but it would be good to include `required` and `aria-required="true"` attributes on required fields.

There are a bunch of other form-related patterns I’d be happy to port over from my work if you are interested.

**Comment by maya** [comment]
@aarongustafson yes, we should definitely add these in! We welcome PRs, but can also add it ourselves too.

Would love to learn more about any other form patterns you think would be useful to include.

Appologies for the delay. 18F is currently at our offsite this week!

**Comment by aarongustafson** [comment]
I can drop in the required bits a little later today. As for other patterns, I have an overview/recap in [my An Event Apart talk "Falling in Love with Forms"](http://www.slideshare.net/AaronGustafson/falling-in-love-with-forms-an-event-apart-dc-2015/). A full rundown of the patterns (for dissection) is [on CodePen](http://codepen.io/aarongustafson/pen/Kppxxy).

**Comment by carodew** [comment]
Thanks @aarongustafson! I'm going to look at these today (just returned home from our 18F offsite). 

Thanks also for talking to our team a few weeks ago. I was watching remotely from Atlanta and I enjoyed the way you graphed the different responsive interactions, and I plan on trying out the technique next time I get the chance.

**Comment by aarongustafson** [comment]
@carodew Awesome!

PS - If you’re in ATL, we should get together some time; I’m just up in Chattanooga. On a side note, we run an event series called [Code & Creativity](http://codeandcreativity.com) up here you might be interested in since you’re so close.

**Comment by carodew** [comment]
@aarongustafson did not realize you were so close, I was in Chattanooga a few weeks ago! Would definitely be up for getting together. LMK if you're ever heading down. 

I will keep my eye on Code & Creativity, sounds cool!

**Comment by maya** [comment]
@aarongustafson were you still planning on submitting a PR or should we do it instead?

**Comment by aarongustafson** [comment]
My apologies @maya, I got caught up in work stuff. Thanks for updating @msecret!

**Comment by maya** [comment]
@aarongustafson no problem! Let us know if you find anything else that needs updating. Thanks again!
